calculator/README.md

```diff
@@ -68,3 +105,31 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+## Authors
+
+👤 **Patrick Ndimukulaga**
+
+- GitHub: [@Kirabo19](https://github.com/Kirabo19)
+- Twitter: [@ndimuku](https://twitter.com/ndimuku)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/patrick-ndimukulaga-a6910a50/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL]] As required by the previous reviewer, make sure you move your readme file to the root of your repository so that it will be shown when someone visits the repository, Kindly address this 👍🏻 
- Note; this should not be an optional requested change, I strongly recommend you make that.